src/pages/guides/sellers/installation-and-varnish-tests.md

```diff
@@ -4,24 +4,24 @@ title: Installation and Varnish tests
 
 # Installation and Varnish tests
```

Comment:
this page needs to be reviewed by the Mercury team to ensure it is accurate related to the testing of the apps.

Reply:
@mikeweis @pdohogne-magento please review.

Reply:
I am not aware of any Varnish tests for apps, unless they are still covered by the existing automated tests.  Installation tests are covered, though not fully automated yet.  The automation checks the npm install, but not any other steps the app may need; those will be covered by manual testing pending further investigation into what can and can't be automated.